### PR TITLE
[WIP] - Upgrade elasticsearch support to 6.2

### LINKED ISSUE #123: latch call seems broken at es 6.2
When trying to use a 6.x ElasticSearch, getting an error regarding the `latch` call. 6.2 notes breaking changes with the returned value on error with incorrect-verb calls. Related?

Relevant code: https://github.com/deconst/content-service/blob/master/src/storage/remote.js#L31-L42
Relevant release note: https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking_60_rest_changes.html#_requests_to_existing_endpoints_with_incorrect_http_verb_now_return_405_responses

cc: @robb-romans

### LINKED ISSUE #127: Upgrade to latest elasticsearch version

### PR DESCRIPTION
**Pull request by SlapBassist**
This includes the necessary changes to support the latest version of Elasticsearch. It required updating the npm package as well as updating how searches are performed.

fixes #127
fixes #123

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "dependencies": {
```
Comment (nimbinatus) [inline package.json:12]: Version bump needed on line 3. As I think this is a release that will break previous versions, this release would bump to `2.0.0`. See https://semver.org/#summary for more info.

**Review by nimbinatus — CHANGES_REQUESTED** [review]
semver update

File: package.json (hunk 1 above)
Comment (SlapBassist) [reply package.json:12]: So we have another PR where the version goes to 1.1.0, and this to 2.0.0, and then we have that mongo bump. What makes sense to version for each of these?

Comment (nimbinatus) [reply package.json:12]: After discussion, decided to go to 2.0.0 for both ES and MongoDB updates (breaking changes, but can go under the same version).

**Review by nimbinatus — APPROVED** [review]

**Pull request merged**